<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_create_measuring_group</class>
 <widget class="QDialog" name="dialog_create_measuring_group">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create measuring group</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonbox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>460</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupbox_measuring_group">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>761</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Measuring group</string>
   </property>
   <widget class="QLabel" name="label_measuring_group_id">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>id:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_measuring_group_id_info">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbutton_measuring_group_load">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Load measuring group</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combobox_measuring_group_load">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_measuring_group_load_description">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load a measuring group.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_measuring_group_description">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>The id of the measure group.</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupbox_measuring_points">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>761</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Measuring stations</string>
   </property>
   <widget class="QTableWidget" name="tablewidget_measuring_point">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>731</width>
      <height>151</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>v2_control_measure_map_id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>table</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>table_id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>weight</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>action</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_measuring_point_field">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>field:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combobox_measuring_point_field">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>initial_waterlevel</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushbutton_measuring_point_add_point">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add station</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_measuring_point_add_point">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add a measuring station to the table.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbutton_measuring_point_load">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Load station</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combobox_measuring_point_load">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_measuring_point_field_description">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>The measure value.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_measuring_point_load_point">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>280</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load a measuring station.</string>
    </property>
   </widget>
  </widget>
  <zorder>groupbox_measuring_points</zorder>
  <zorder>buttonbox</zorder>
  <zorder>groupbox_measuring_group</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonbox</sender>
   <signal>accepted()</signal>
   <receiver>dialog_create_measuring_group</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonbox</sender>
   <signal>rejected()</signal>
   <receiver>dialog_create_measuring_group</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
